lib/wasi/tests/runners.rs

```diff
@@ -68,6 +67,8 @@ mod wasi {
         });
         let err = handle.join().unwrap().unwrap_err();
 
+        dbg!(&err);
+
```

Comment:
Remove me?

Reply:
Done. That `dbg!()` was added temporarily while troubleshooting so I could see which error we were getting. I've adjusted the assertion so we print out unknown error kinds.